api/mapping/views.py

```diff
@@ -1147,3 +1155,54 @@ def get_context_data(self, **kwargs):
             json_response = get_json_from_nlpmodel(json=ast.literal_eval(query.json_response))
             context = {"user_string": query.user_string, "results": json_response}
             return context
+
+
+def save_scan_report_value_concept(request):
+    if request.method == "POST":
+        form = ScanReportValueConceptForm(request.POST)
+        if form.is_valid():
+
+            scan_report_value = ScanReportValue.objects.get(
+                pk=form.cleaned_data['scan_report_value_id']
+            )
+
+            try:
+                concept = Concept.objects.get(
+                    concept_id=form.cleaned_data['concept_id']
+                )
+            except Concept.DoesNotExist:
+                messages.error(request,
+                                 "Concept id {} does not exist in our database.".format(form.cleaned_data['concept_id']))
+                return redirect("/values/?search={}".format(scan_report_value.scan_report_field.id))
+
+            scan_report_concept = ScanReportConcept.objects.create(
+                concept_name=concept.concept_name,
+                concept_id=concept.concept_id,
+                entity="empty",
```

Comment:
Do we want to hardcode this as "empty" or leave the field null/blank?

Reply:
Good point. I am not sure we need this attribute in the model. How are we using it?

Reply:
These fields are used to save the return from NLP. e.g. 'entity' could be DrugAdministered or SignOrSymptom.